universal-application-tool-0.0.1/test/views/components/FieldWithLabelTest.java

```diff
@@ -38,7 +38,7 @@ public void createEmail_rendersEmailInput() {
   @Test
   public void number_setsNoValueByDefault() {
     FieldWithLabel fieldWithLabel = FieldWithLabel.number();
-    assertThat(fieldWithLabel.getContainer().render()).doesNotContain("value");
+    assertThat(fieldWithLabel.getContainer().render()).doesNotContain(" value");
```

Comment:
question: Why add a space here?

Reply:
The intent here is to check there is no value set for the input element and we check by a substring match. This matches with the new `this.value` in `onintput` attribute. Adding a space is a workaround for that. Agreed it is not obvious, I can add some comments to explain.

Reply:
Ohh like no `value="<value>"` set?

Reply:
Yes.